services/hanke-service/build.gradle.kts

```diff
@@ -5,7 +5,7 @@ import org.springframework.boot.gradle.tasks.run.BootRun
 group = "fi.hel.haitaton"
 version = "0.0.1-SNAPSHOT"
 java.sourceCompatibility = JavaVersion.VERSION_11
-val postgreSQLVersion = "42.2.18"
+val postgreSQLVersion = "42.6.0"
```

Comment:
I think it is possible to let spring decide the compatible postgresql version. I.e. this can be removed, if there is no strict need for us to use a specific version.

Reply:
Good catch. Removed the version from the dependency.

I checked the other dependencies and noticed that there was an unnecessary declaration:
```
testImplementation("org.junit.jupiter:junit-jupiter:5.8.1")
```
`spring-boot-starter-test´ already imports that one.

Also `mockwebserver` has a managed version in Spring Boot, but in 2.6 it's a lot older than the one we use. 2.7 updates that dependency, so I'll remove the version in that PR.